src/main/java/com/cloudbees/plugins/credentials/util/RestrictiveEntityResolver.java

```diff
@@ -0,0 +1,32 @@
+package com.cloudbees.plugins.credentials.util;
+
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.xml.sax.EntityResolver;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import java.io.IOException;
+
+/**
+ * This class is a clone of {@link jenkins.util.xml.RestrictiveEntityResolver} because the last is Restricted. If we
+ * ever unrestrict it, we should remove this class and use the first one.
```

Comment:
So, file a core PR to unrestrict it, since clearly plugins need it.

Reply:
We already did

Reply:
https://github.com/jenkinsci/jenkins/pull/4032, I see. So amend this comment to include the magic `TODO` keyword and specify the (weekly) version containing that API (https://github.com/jenkinsci/jenkins/pull/4032#discussion_r287387863). That way future maintainers can just grep for TODOs and quickly see if there is something actionable.

Reply:
done